scheduler/crud/flyte_scheduler.go

```diff
@@ -0,0 +1,89 @@
+package crud
```

Comment:
what does crud mean?
also 
`scheduler/crud/flyte_scheduler.go` is weird
how about
`scheduler/api.go` -> just the interface
then `scheduler/db_api/api_impl.go`